src/components/badges/_badges.scss

```diff
@@ -13,12 +12,13 @@ $include-html: false !default;
 
   .mint-badge {
     @include component;
+    @include fixText($badgeFontSize);
+    overflow: visible; // allow badges align as text
+    min-height: auto;
+    height: auto;
```

Comment:
Yeah, they are: `min-height` is needed for overriding `min-height` from `component`.
`height` overrides the value from `fixText`